src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String readString = readFile(fromFileName);
+        if (!readString.isEmpty()) {
+            int supplySum = 0;
+            int buySum = 0;
+
+            String[] stringInput = readString.split("\\W+");
+            for (int i = 0; i < stringInput.length / 2; i++) {
+                if (stringInput[i * 2].equals("supply")) {
+                    supplySum += Integer.parseInt(stringInput[i * 2 + 1]);
+                } else if (stringInput[i * 2].equals("buy")) {
+                    buySum += Integer.parseInt(stringInput[i * 2 + 1]);
+                }
+            }
+            String statistic = generateStatistic(supplySum, buySum);
```

Comment:
```suggestion
        if (!readString.isEmpty()) {
            int supplySum = 0;
            int buySum = 0;

            String[] stringInput = readString.split("\\W+");
            for (int i = 0; i < stringInput.length / 2; i++) {
                if (stringInput[i * 2].equals("supply")) {
                    supplySum += Integer.parseInt(stringInput[i * 2 + 1]);
                } else if (stringInput[i * 2].equals("buy")) {
                    buySum += Integer.parseInt(stringInput[i * 2 + 1]);
                }
            }
            String statistic = generateStatistic(supplySum, buySum);
```

move this to the createReport method